models/commentModel.js

```diff
@@ -0,0 +1,20 @@
+const db = require("../db/connection");
+/** Imports & BP above here */
+
+exports.selectCommentsByArticleId = (id) => {
+  return db
+    .query(
+      `SELECT comment_id, author, votes, created_at, body, article_id
+        FROM comments
+        WHERE article_id = $1
+        ORDER BY created_at DESC;`,
+      [id]
+    )
+    .then((result) => {
```

Comment:
Have a think about why the query would be returning empty rows and how we would handle this in different ways. 
a) article does not exist 
b) article exists but has no comments - would this be an error? What would you send back instead?